daemon/daemon.go

```diff
@@ -373,6 +374,21 @@ func (daemon *Daemon) restore(cfg *configStore) error {
 						Type: local.Name,
 					}
 				}
+
+				// Normalize the "default" network mode into the network mode
+				// it aliases ("bridge on Linux and "nat" on Windows). This is
+				// also done by the container router, for new containers. But
+				// we need to do it here too to handle containers that were
+				// created prior to v26.0
+				//
+				// TODO(aker): remove this migration code in v27.0.
```

Comment:
I think we need to have a discussion around when to remove data migrations given the increased pace of major releases and having users which stick to LTM releases.

Reply:
Good call.

I thought we did only support upgrades from one major to the next one but I fail to find anything about that in the docs. If that's the case we can stick to v27.0 in this specific instance.

Reply:
I changed that to:

```
// TODO(aker): remove this migration code once the next LTM version of MCR is released.
```